python/sunbeam/deck/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+
+"""
```

Comment:
There seems to be something missing between the first `"""` and the second `"""`